lib/Graph/Nodes.cpp

```diff
@@ -936,10 +936,14 @@ bool ExpNode::verify() const {
   const Node *parent = getResult().getNode();
   bool isValid =
       checkSameIsQuantized(getInput().getType(), getResult().getType(), parent);
+
   if (getInput().getType()->isQuantizedType()) {
-    return false;
+    isValid &= checkType(getInput(), getResult().getElementType(),
+                         getResult().getNode());
+  } else {
+    isValid &= checkSameType(getInput(), getResult(), parent);
```

Comment:
nit: This check includes shape checking, right? So perhaps we move the checkSameShape below into the if above.

Reply:
Good point, moved.